lib/rubygems/commands/query_command.rb

```diff
@@ -9,6 +9,13 @@ class Gem::Commands::QueryCommand < Gem::Command
 
   include Gem::QueryUtils
 
+  NAME    = 'query'
+  SUMMARY = <<~SUMMARY
+    Query gem information in local or remote repositories
+
+    **Note:** `query` is deprecated as of RubyGems 3.2
+  SUMMARY
```

Comment:
Summaries should be one liners, otherwise `gem help commands` output is a bit messed up:

```
$ gem help commands
GEM commands are:

    build                  Build a gem from a gemspec
    cert                   Manage RubyGems certificates and signing settings
    check                  Check a gem repository for added or missing files
    cleanup                Clean up old versions of installed gems
    contents               Display the contents of the installed gems
    dependency             Show the dependencies of an installed gem
    environment            Display information about the RubyGems environment
    fetch                  Download a gem and place it in the current directory
    generate_index         Generates the index files for a gem server directory
    help                   Provide help on the 'gem' command
    info                   Show information for the given gem
    install                Install a gem into the local repository
    list                   Display local gems whose name matches REGEXP
    lock                   Generate a lockdown list of gems
    mirror                 Mirror all gem files (requires rubygems-mirror)
    open                   Open gem sources in editor
    outdated               Display all gems that need updates
    owner                  Manage gem owners of a gem on the push server
    pristine               Restores installed gems to pristine condition from
                           files located in the gem cache
    push                   Push a gem up to the gem server
    query                  Query gem information in local or remote repositories
                           
                           **Note:** `query` is deprecated as of RubyGems 3.2
    rdoc                   Generates RDoc for pre-installed gems
    regenerate_binstubs    Re run generation of executable wrappers for gems.
    search                 Display remote gems whose name matches REGEXP
    server                 Documentation and gem repository HTTP server
    signin                 Sign in to any gemcutter-compatible host. It defaults
                           to https://rubygems.org
    signout                Sign out from all the current sessions.
    sources                Manage the sources and cache file RubyGems uses to
                           search for gems
    specification          Display gem specification (in yaml)
    stale                  List gems along with access times
    uninstall              Uninstall gems from the local repository
    unpack                 Unpack an installed gem to the current directory
    update                 Update installed gems to the latest version
    which                  Find the location of a library file you can require
    yank                   Remove a pushed gem from the index

For help on a particular command, use 'gem help COMMAND'.

Commands may be abbreviated, so long as they are unambiguous.
e.g. 'gem i rake' is short for 'gem install rake'.
```

Reply:
Good call! Sorry for the miss on this 😬 

I'm happy to make adjustments here. I think they'll depend on the direction you want to take this. A couple alternatives that make sense to me:
* `gem query`'s summary becomes `Query gem information in local or remote repositories **deprecated**`
* The deprecation message is moved to `#description`
* The deprecation is removed altogether in favor of not rendering documentation for deprecated commands
  * Should `gem help commands ` be consistent with the guides and skip deprecated commands?
  * Should there be mention of the deprecation anywhere other than in CLI output upon use?

Please let me know the direction you'd like this to take and I'll follow suit 😄